synfig-studio/src/gui/CMakeLists.txt

```diff
@@ -19,6 +19,21 @@ add_definitions(-DSYNFIG_DATADIR="${SYNFIG_DATA_PREFIX}")
 add_definitions(-DLOCALEDIR="${DATA_PREFIX}/share/locale")
 add_definitions(-DCMAKE_BUILD)
 
+if (CMAKE_BUILD_TYPE MATCHES Debug)
+    add_definitions("-DSHOW_EXTRA_INFO")
+
+    execute_process(
```

Comment:
Since this code is not portable and cannot be run under Windows (without MSYS), it is better to manually set `ETL_VERSION="1.5.0"` and move this code to the main CMakeLists.txt